pkg/dto/response.go

```diff
@@ -248,12 +248,45 @@ type ScanVulnerabilityDetailResponse struct {
 	VendorComments []tools.VendorComment `json:"vendor_comments"`
 	References     []string              `json:"references"`
 
+	Metrics []CVSSMetric `json:"metrics"`
+
+	CWERemediation []CWERemediation `json:"remediation"`
```

Comment:
It would make sense for this to be plural, wouldn't it?

```suggestion
	CWERemediations []CWERemediation `json:"remediation"`
```